python/src/aiconfig/editor/server/server.py

```diff
@@ -330,6 +331,11 @@ def run() -> FlaskResponse:
         "callback_manager": True,
     }
     state = get_server_state(app)
+
+    # Allow user to modify their environment keys without reloading the server.
+    # Execution time of `0.001s` is arbitrary, but should be small enough to not be noticeable.
+    dotenv.load_dotenv()
```

Comment:
We should consider just adding these to model parsers directly, but this moves us forwards and we can change that later if we want